src/annotator/adder.tsx

```diff
@@ -1,38 +1,38 @@
 import { render } from 'preact';
 
 import AdderToolbar from './components/AdderToolbar';
+import type { Command } from './components/AdderToolbar';
 import { isTouchDevice } from '../shared/user-agent';
+import type { Destroyable } from '../types/annotator';
+
 import { createShadowRoot } from './util/shadow-root';
 
 /**
- *  @typedef {1} ArrowPointingDown
  * Show the adder above the selection with an arrow pointing down at the
  * selected text.
  */
 export const ARROW_POINTING_DOWN = 1;
 
 /**
- *  @typedef {2} ArrowPointingUp
  * Show the adder above the selection with an arrow pointing up at the
  * selected text.
  */
 export const ARROW_POINTING_UP = 2;
 
-/**
- *  @typedef {ArrowPointingDown|ArrowPointingUp} ArrowDirection
- * Show the adder above the selection with an arrow pointing up at the
- * selected text.
```

Comment:
I have removed this comment from the `ArrowDirection` type, because it was the same as the one in `ArrowPointingUp`. I'm not sure if that was intended.